canonicalwebteam/image_template/__init__.py

```diff
@@ -11,7 +11,7 @@
 template = env.get_template("image_template.html")
 
 
-def image_template(url, alt, width, height, hi_def, **attributes):
+def image_template(url, alt, width, height, hi_def, lazy, **attributes):
```

Comment:
you can do `..., hi_def, lazy=True, **attributes):` for a default value.

Reply:
I see what you're saying now regarding required positional attribute.

For this you can remove the `lazy` all together, `**attributes` will catch all other params passed in.